components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -343,6 +343,9 @@ public void close() throws IOException {
     if (in != null) {
       in.close();
     }
+    if (tiffSaver != null) {
+      tiffSaver.close();
```

Comment:
I don't think I would feel comfortable fiddling with finalization at this stage.